### DISCUSSION TITLE
constant sync between 2 devices

### DISCUSSION THREAD

**Discussion by wieluk** [mtkennerly/ludusavi#336] [General]
Is it possible to have a constant sync on two devices?
How would I do it? 

Best way would be to invoke a backup on file change.
Right now I just have it scheduled on logon and every hour.

But I have to restore if the other computer changes the savefile right?
Is this possible?

**Comment by mtkennerly** [comment]
Hi! There are two parts to this:

* Automatic sync between devices, without manually using the upload/download buttons in Ludusavi. For this, you could use something like [Syncthing](https://syncthing.net/).
* Automatic restore before playing and backup after playing. For this, you could configure a [wrapper command](https://github.com/mtkennerly/ludusavi?tab=readme-ov-file#game-launch-wrapping) in your launcher (e.g., Steam: game properties -> launch options -> `C:\...\ludusavi.exe wrap --infer steam --gui -- %command%`). You could also use a launcher with Ludusavi integration, like [Playnite](https://playnite.link/) with the [Ludusavi plugin](https://github.com/mtkennerly/ludusavi-playnite).
